theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/CurveEditorPopover/SVGCurveSegment.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+const SVG_PADDING = 1
+const SVG_CIRCLE_RADIUS = 0.1
+
+const CURVE_COLOR = '#858585'
+const CONTROL_COLOR = '#4f4f4f'
+const CONTROL_HITZONE_COLOR = 'rgba(255, 255, 255, 0.1)'
+
+type IProps = {
+  easing: [number, number, number, number] | null
+}
+
+const SVGCurveSegment: React.FC<IProps> = (props) => {
+  const {easing} = props
+  if (easing) {
+    return (
+      <svg
+        height="100%"
+        width="100%"
+        viewBox={`${-SVG_PADDING} ${-SVG_PADDING} ${1 + SVG_PADDING * 2} ${
+          1 + SVG_PADDING * 2
+        }`}
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <line
+          x1="0"
+          y1="1"
+          x2={easing[0]}
+          y2={1 - easing[1]}
```

Comment:
Change: Please name these variables to create self describing code at the top of this file–separate from the JSX declaration.